perllib/FixMyStreet/App/Controller/Waste.pm

```diff
@@ -859,6 +859,36 @@ sub request : Chained('property') : Args(0) {
             intro => 'request/intro.html',
             check_unique_id => 0,
             next => $next,
+            update_field_list => sub {
```

Comment:
I see why this is here, and works fine, but I guess ideally we'd want it not to be, given it's Bexley specific :)
Wonder how best to do that. I guess we have the cobrand available in $form->{c} here (I think), so you could move this to a cobrand call_hook `munge_waste_request_page` or something like that, and then call it from here?

Reply:
Yeah, this wasn't going to be its permanent place, I need to refactor some things still but wanted to get something working :-) Yeah, I was thinking a call_hook would do the job in some way.